lib/views/request/request_view.dart

```diff
@@ -128,13 +128,35 @@ class _RequestViewState extends State<RequestView> {
           display = display.substring(0, display.length - 1);
         }
 
+        /// when we first boot up the request screen
+        if(display == '0') {
+          setState(() {
+            display = value;
+          });
+          return;
+        }
+
+        // if(display.startsWith('.')) {
+        //   setState(() {
+        //     display = '0.';
+        //   });
+        //   return;
+        // }
+
```

Comment:
mh some of this code do not make sense to me, or maybe I am missing somethings, but I spend some time in this commit https://github.com/dart-lightning/lndart.clnapp/commit/f57cf5d1dabdec4ec624113d3cf3386c023202d4 to organize it

So you can do just `git cherry-pick f57cf5d1dabdec4ec624113d3cf3386c023202d4`? and work on my code that it  is already well tested?